lock/dynamolock/dynamolock_test.go

```diff
@@ -41,34 +42,77 @@ func (m *mockDynamoDBClient) UpdateItemWithContext(ctx context.Context, input *d
 }
 
 func TestLock(t *testing.T) {
-
 	client := &mockDynamoDBClient{}
-	options := LockOptions{
+	options := Options{
 		TTL:       2 * time.Second,
 		HeartBeat: 10 * time.Millisecond,
 	}
-	lockObj, err := New(client, "delta_lock_table", "_commit.lock", options)
-
+	dl, err := New(client, "delta_lock_table", "_commit.lock", options)
 	if err != nil {
 		t.Error("error occurred.")
 	}
-	haslock, err := lockObj.TryLock()
+
+	haslock, err := dl.TryLock()
 	if err != nil {
 		t.Error("error occurred.")
 	}
 	if haslock {
 		t.Logf("Passed.")
 	}
-	lockObj.Unlock()
+
+	err = dl.Unlock()
+	if err != nil {
+		t.Error(err)
+	}
 
 	//TODO Check into why the lock is expired before one second.
```

Comment:
regarding this existing TODO - the lock is expired because the unlock call immediately before it releases the lock, which counts as expired.  So any of the dynamolock tests that call unlock and then test for expiry aren't actually testing the expiry time.